core/cmd/client.go

```diff
@@ -284,6 +284,11 @@ type ChainlinkRunner struct{}
 // Run sets the log level based on config and starts the web router to listen
 // for input and return data.
 func (n ChainlinkRunner) Run(app chainlink.Application) error {
+	defer func() {
+		if err := app.GetLogger().Sync(); err != nil {
+			panic(err)
```

Comment:
Is there a different way to report this that won't mask an existing panic?

Reply:
Good point. Not easily, I'll have to fallback to the system logger.